src/assets/translations/main.json

```diff
@@ -168,7 +168,9 @@
         "foxyBalance": "FOXy Balance",
         "withdrawals": "Withdrawals",
         "emptyWithdraws": "Pending or active withdrawals will show here.",
-        "availableDate": "available %{date}"
+        "availableDate": "available %{date}",
+        "approveToken": "Approve",
```

Comment:
Maybe could be named "approve" ? since the string does not contain any token mention

Reply:
Yeah, I'd say you could call it `approve` and move it into the common area.